lib/index.coffee

```diff
@@ -13,7 +13,10 @@ CLOSE_MESSAGE = "Thrift-pool: Connection closed"
 #  @param thrift_options, passed to thrift connection,
 create_cb = (thrift, pool_options, thrift_options, cb) ->
   cb = _.once cb
-  connection = thrift.createConnection pool_options.host, pool_options.port, thrift_options
+  if pool_options.ssl?
```

Comment:
If you make this a boolean you can do `pool_options.ssl ?= false` and then just check `if pool_options.ssl`